toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -232,7 +236,7 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	}
 
 	if len(rpmConflicts) != 0 {
-		logger.Log.Error("RPM Conflicts with toolchain:")
+		logger.Log.Error("RPM Conflicts with toolchain (ignored if ALLOW_TOOLCHAIN_REBUILDS=y): ")
```

Comment:
Not part of your changes, but since we're here, could you spell "conflicts" with small caps?